src/graphql/schema/collection/createCollectionRelationTailGqlFieldEntries.ts

```diff
@@ -46,11 +46,12 @@ export default function createCollectionRelationTailGqlFieldEntries <TSource, TV
             : formatName.field(`${headCollection.type.name}-by-${relation.name}`),
           {
             description: `Reads a single ${scrib.type(headCollectionGqlType)} that is related to this ${scrib.type(collectionGqlType)}.`,
+            externalFieldNameDependencies: relation._tailFieldNames, // 🔥 Needs to account for classicIds
```

Comment:
Why do we need to account here for classicIds? The source of a relation is a column != id , it does not seem to address the destination of the relation here.